source/agora/common/DNS.d

```diff
@@ -955,7 +955,7 @@ public struct OPTRR
     ***************************************************************************/
 
     public ResourceRecord record = ResourceRecord(
-        Domain.init, TYPE.OPT, cast(CLASS) 4096, 0, ResourceRecord.RDATA(null));
+        Domain.init, TYPE.OPT, cast(CLASS) 2048, 0, ResourceRecord.RDATA(null));
```

Comment:
Would be nice to also define the buffer in terms of `OPTRR.init.payloadSize()` so that we don't have to keep those values in sync. And I think 4096 is more common.